FFXIVClientStructs/FFXIV/Client/Game/UI/PlayerState.cs

```diff
@@ -25,6 +25,7 @@ public unsafe partial struct PlayerState {
     [FieldOffset(0x7D)] public byte Tribe;
     [FieldOffset(0x7E)] public byte CurrentClassJobId;
 
+    [FieldOffset(0x80)] public void* CurrentClassJobRow;
```

Comment:
Should this be nint? I believe we want to avoid void* where possible

Reply:
I don't mind either. I just thought I'd use void* because we don't have row structs.
Changed it to nint.